File: DAY2/FindTheDulplicateElement.c
int findDuplicate(int* nums, int numsSize){
    int aux[numsSize-1];

    for(int i = 0; i < numsSize-1; i++) 
        aux[i] = 0;

    for(int i = 0; i <= numsSize; i++) {
        aux[nums[i]-1] += 1;
        if (aux[nums[i]-1] > 1)
            return nums[i];
    }
    return 0;
}

File: DAY2/FindTheDulplicateElement.java
class Solution {
    public int findDuplicate(int[] nums) {
        int slow = nums[0];
        int fast = nums[0];
        do{
            slow = nums[show];
            fast = nums[nums[fast]];
        }
        while(slow!=fast);
        fast = nums[0];
        while(slow!=fast)
        {
            slow = nums[0];
            fast = nums[0];
        }
        return slow;
    }
}
